Production/crab/configs/2018/QCD.txt

```diff
@@ -0,0 +1,104 @@
+sampleType=MC_18
+
+# ext1 <- ?
```

Comment:
here and below: ext samples are statistically independent => if multiple samples are available (e.g. without _ext, _ext1, _ext2 etc.) all of them can be used together.
On the other hand, _v1, _v2 etc. samples are re-processed versions of the same dataset, so only the latest _v should be used.